server/edusync/src/main/java/com/codestates/edusync/model/study/studygroupjoin/controller/StudygroupJoinController.java

```diff
@@ -87,7 +87,7 @@ public ResponseEntity deleteStudygroupJoin(@PathVariable("studygroup-id") @Posit
                                                Authentication authentication) {
         Member loginMember = memberVerificationService.getLoggedIn(authentication);
 
-        studygroupJoinService.deleteStudygroupJoin(studygroupId, loginMember);
+        studygroupJoinService.deleteApprovedMemberSelf(studygroupId, loginMember);
```

Comment:
```suggestion
        studygroupJoinService.deleteMemberSelf(studygroupId, loginMember);
```

이것도 어차피 멤버 탈퇴니까 저렇게 보는게 깔끔해 보여요

Reply:
넵. 위쪽이랑 공통사항이네요.
이름 변경하시면 될 것 같습니다 !

Reply:
저도 기능이 구분되지 않는 한 이게 더 좋아보이네요!